src/main/java/cs2103/v15_1j/jimjim/command/SaveLocationCommand.java

```diff
@@ -23,11 +23,13 @@ public UIFeedback execute(ControllerStates conStates) {
     	conStates.config.savePath = savePath;
     	
     	if (conStates.storage.saveConfig(conStates.config)) {
+    		conStates.storage.setSaveFile(savePath);
```

Comment:
need to save the data to the new location also. Hence you should move this to before the if else, and add to the if condition to check that saving the current data to the new location succeeds. Please add a test case for trying to save to an invalid path and see whether the program handles it correctly as well.